addons/sale_quotation_builder/models/sale_order.py

```diff
@@ -42,21 +42,19 @@ class SaleOrderLine(models.Model):
 
     website_description = fields.Html('Website Description', sanitize=False, translate=html_translate, sanitize_form=False)
 
-    @api.model
-    def create(self, values):
-        values = self._inject_quotation_description(values)
```

Comment:
these helpers are typically overriden by community modules to easily customise the behaviour, always nice to keep